docs/cep/source/source-mapping/index.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 1
+title: Source Mapping
+---
+
+Each `source.type` configuration can have a mapping denoted by the `map.type` annotation that defines how to convert the incoming event format to Stream events.
+
+The `type` parameter of the `map.type` defines the map type to be used in converting the incoming events. The other parameters of `map.type` annotation depends on the mapper selected, and some of its parameters can be optional.
+
+For detailed information about the parameters see the documentation of the relevant mapper.
+
+### Map Attributes
```

Comment:
@StoyanTc, this section about Map Attributes does not seem correct to me, considering we changed annotations to WITH, according to the Migration Guide.

Reply:
@StoyanTc, any idea what's up with these map attributes?